.gitignore

```diff
@@ -13,3 +13,9 @@ src/config.mlh
 .direnv
 .rustup
 result
+# precompiled bindings
+src/bindings/compiled
+# allows dune and README.md
+src/bindings/mina-transaction/gen/*/*.js
+src/bindings/mina-transaction/gen/*/*.ts
```

Comment:
is there a reason to ignore these files? they're small and the diff lets you easily see type changes, which is convenient when reviewing

Reply:
I just want to stop tracking generated files in git entirely.

I could write a script to download them for a pair of commits and show the diff if that would help.

Reply:
personally I would track them. they are much more "part of the codebase" than the bindings files. for example, the TS build won't even work without those files, while bindings are only needed once you compile circuits/create proofs

Reply:
Fair enough :+1: